src/components/elements/maps/MultiGeolocationMap.tsx

```diff
@@ -0,0 +1,70 @@
+import L, { LatLngExpression } from 'leaflet';
+import { useMemo } from 'react';
+import { FeatureGroup, Marker, Popup } from 'react-leaflet';
+
+import BaseMapContainer, { BaseMapContainerProps } from './BaseMapContainer';
+import { GeolocationFieldsWithMetadataFragment } from '@/types/gqlTypes';
+
+interface Props {
+  geolocations: GeolocationFieldsWithMetadataFragment[];
+  renderMarkerContent?: (
+    geolocation: GeolocationFieldsWithMetadataFragment
+  ) => React.ReactNode;
+  height: string | number;
+  BaseMapContainerProps?: Omit<BaseMapContainerProps, 'children' | 'height'>;
+}
+
+const locationToLatLngExpression = (
+  location: GeolocationFieldsWithMetadataFragment
+): LatLngExpression => {
+  return [
+    parseFloat(location.coordinates.latitude),
+    parseFloat(location.coordinates.longitude),
+  ] as LatLngExpression;
+};
+
+// Map that displays multiple Geolocations as markers
+const MultiGeolocationMap: React.FC<Props> = ({
+  geolocations,
+  renderMarkerContent,
+  height,
+  BaseMapContainerProps,
+}) => {
+  // find bounds for the map
+  const bounds = useMemo(() => {
+    const latLngExpressions: LatLngExpression[] = geolocations.map((loc) =>
+      locationToLatLngExpression(loc)
+    );
+    return L.latLngBounds(latLngExpressions);
+  }, [geolocations]);
+
+  return (
+    <BaseMapContainer
```

Comment:
The SingleGeolocationMap gets a key prop, I assume to force re-render. Do we need anything like that here?

Reply:
The key was not necessary on the SingleGeolocationMap map, I removed it. I think it was from a previous iteration but I confirmed it renders as expected without. THanks!